src/main/java/org/cascadebot/cascadebot/utils/votes/VoteButtonGroup.java

```diff
@@ -65,6 +82,55 @@ public void addVote(User user, Object vote) {
                 votes.remove(user.getIdLong());
                 return;
             }
+        } else {
+            DecimalFormat df = new DecimalFormat("#");
+            df.setRoundingMode(RoundingMode.HALF_UP);
+            long timeElapsed = Instant.now().toEpochMilli() - timerStartTime;
+            int elapsed = Integer.parseInt(df.format(timeElapsed /1000));
+            int newTimersTime;
+
+            if (timerRunTime < 30 && !user.isBot() && user.getIdLong() != getOwnerId()) {
+                if ((timerRunTime + 5) > 30) {
+                    timerRunTime = 30;
+                    newTimersTime = timerRunTime - elapsed;
+                } else {
+                    timerRunTime += 5;
+                    newTimersTime = (timerRunTime - elapsed);
+                }
+
+                voteTimer.cancel();
+                voteTimer = new Timer();
+                voteTimer.schedule(new TimerTask() {
+                    @Override
+                    public void run() {
+                        CascadeBot.INS.getShardManager().getGuildById(getGuildId()).getTextChannelById(getChannelId()).retrieveMessageById(getMessageId()).queue(message -> {
+                            message.delete().queue(null, DiscordUtils.handleExpectedErrors(ErrorResponse.UNKNOWN_MESSAGE));
+                        }, DiscordUtils.handleExpectedErrors(ErrorResponse.UNKNOWN_MESSAGE));
+                        voteFinished();
+                        finishConsumer.accept(getOrderedVoteResults());
+                    }
+                }, TimeUnit.SECONDS.toMillis(newTimersTime));
+
+                List<Guild> mutualGuilds = user.getMutualGuilds();
+                int position = 0;
+                for (int i = 0; i < mutualGuilds.size(); i++){
+                    if (mutualGuilds.get(i).getIdLong() == getGuildId()){
+                        position = i;
+                    }
+                }
+                Member member = mutualGuilds.get(position).getMember(user);
```

Comment:
You could do `CascadeBot.INS.getShardManager().getGuildById(getGuildId()).getMember(user)`